functions/notify_slack.py

```diff
@@ -202,6 +222,13 @@ def default_notification(subject, message):
 def filter_message_from_slack(message):
     if message.get('source', "") == "aws.iam" and message.get('detail', {}).get('eventName', '') in ["GenerateCredentialReport", "GenerateServiceLastAccessedDetails", "CreateServiceLinkedRole"]:
       return True
+    elif message.get('source', "") == "aws.ecr":
+      if message.get('detail', {}).get('finding-severity-counts', {}).get('CRITICAL', "") != 0:
+        return False
+      elif message.get('detail', {}).get('finding-severity-counts', {}).get('HIGH', "") != 0:
```

Comment:
I think I am going to remove alerting on `HIGH` vulns... currently every single image nearly has them, mainly a `busybox` vulnerability. @rraub What are your thought's?  Should we still go for only notifying on `CRITICAL` ? This will be very chatty for QA deployments